### PR TITLE
Only natural-ish haircolors are generated randomly

### PR DESCRIPTION
**Pull request by Jack7D1**
## About The Pull Request
Simply adds a new proc that returns a RGB value between 0x00 0x00 0x00 and 0xFF 0x80 0x00
Applies this proc to the create_mob proc, which is used for ghost role spawners and generally spawned humans.
This does not alter nonhuman species or character creation in the creation panel at all.

## Why It's Good For The Game
Less terrible hodge-podge skin-hair combinations that are horrible to look at from ghost roles and random spawns.

## Changelog
:cl:
tweak: Randomly generated characters now have less neon colored hairs
/:cl:

### CODE REVIEW

**Comment by triplezeta** [comment]
speedmerge

File: code/__HELPERS/text.dm [hunk 1]
```diff
@@ -84,7 +84,7 @@ GLOBAL_LIST_INIT(markup_regex, list("/"  = new /regex("((\\W|^)_)(\[^_\]*)(_(\\W
 	for(var/tag in (GLOB.markup_tags - ignore))
 		markup = GLOB.markup_regex[tag]
 		message = markup.Replace_char(message, "$2[GLOB.markup_tags[tag][1]]$3[GLOB.markup_tags[tag][2]]$5")
-
```
Comment (Jack7D1) [inline code/__HELPERS/text.dm:87]: I don't know what did this but hey, less weird whitespace I guess

File: code/__HELPERS/text.dm [hunk 2]
```diff
@@ -84,7 +84,7 @@ GLOBAL_LIST_INIT(markup_regex, list("/"  = new /regex("((\\W|^)_)(\[^_\]*)(_(\\W
 	for(var/tag in (GLOB.markup_tags - ignore))
 		markup = GLOB.markup_regex[tag]
 		message = markup.Replace_char(message, "$2[GLOB.markup_tags[tag][1]]$3[GLOB.markup_tags[tag][2]]$5")
-	
+
 	return message
```
Comment (Jack7D1) [inline code/__HELPERS/text.dm:88]: I don't know what did this

**Comment by Jack7D1** [comment]
effectively a port of https://github.com/tgstation/tgstation/pull/54949

**Comment by Maldaris** [comment]
> effectively a port of [tgstation/tgstation#54949](https://github.com/tgstation/tgstation/pull/54949)

Do we link that in the PR even though you're the author in both places 🤔

**Comment by triplezeta** [comment]
> > effectively a port of [tgstation/tgstation#54949](https://github.com/tgstation/tgstation/pull/54949)
> 
> Do we link that in the PR even though you're the author in both places 🤔

ye

**Comment by Superyodeler** [comment]
The TG PR got closed.

**Comment by Jack7D1** [comment]
based

**Comment by Sparrowbird611** [comment]
Praise God

**Review by MarkSuckerberg — APPROVED** [review]
F I N A L L Y

**Pull request merged**